ops/aws/canary/lambda/pkg/scenarios/submitDockerEstuaryJobAndGet.go

```diff
@@ -0,0 +1,111 @@
+package scenarios
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/filecoin-project/bacalhau/pkg/downloader"
+	"github.com/filecoin-project/bacalhau/pkg/downloader/util"
+	"github.com/filecoin-project/bacalhau/pkg/model"
+	"github.com/filecoin-project/bacalhau/pkg/system"
+	"github.com/rs/zerolog/log"
+)
+
+// This test submits a job that uses the Docker executor with an IPFS input.
+func SubmitDockerEstuaryJobAndGet(ctx context.Context) error {
```

Comment:
We don't want issues with Estuary to be triggering our alerting when there's nothing we can do about it. Writing and reading from Estuary should always be on a "best effort" basis with a fallback if Estuary is not available.

...I think this canary job can't actually test this? Because the Estuary write will sometimes/often return an IPFS result from our own node, and then the Estuary download should also have an IPFS back up?

Reply:
Ignore me – I think this is valuable. Jobs that explicitly target Estuary should not be failing under any circumstances.